12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,470 @@
+---
+description: Get started with the Content Delivery API.
+---
+
+# Content Delivery API
+
+## Overview
```

Comment:
Do we need this heading? Seems a bit redundant, like, we do not really need to know that this is an overview 🤔 
What do you think?

Reply:
I agree 😊 I'll remove it